Automattic/pocket-casts-ios
# Search Improvements: show podcasts and episodes

| 📘 Project: #709 |
|:---:|

Display the podcasts and episodes when performing a search.

## To test

1. Run the app
2. Go to Profile > Settings > Beta Features > enable `newSearch`
3. Change the app Build Configuration to `Staging`
3. Go to Discover ¶
4. Tap the search input
5. Enter any term
6. ✅ Podcasts and episodes should reflect what you searched
7. Change your search term
8. ✅ The results should change
9. Type any random string
10. ✅  You should see a message that no episodes or podcasts were found

¶ The search is not 100% functional from Podcasts and this will be fixed in a subsequent PR. I avoided doing that now because it would affect the current search.

## Checklist
